integrations/jetstream/documentation.yaml

```diff
@@ -0,0 +1,39 @@
+exporter_type: included
+app_name_short: JetStream
+app_name: {{app_name_short}}
+app_site_name: {{app_name_short}}
+app_site_url: https://github.com/AI-Hypercomputer/JetStream
+exporter_name: {{app_name_short}}
+exporter_repo_url: https://github.com/AI-Hypercomputer/JetStream/blob/main/docs/observability-prometheus-metrics-in-jetstream-server.md
```

Comment:
Forgot about this: We should also add a variable for the GKE setup doc:

gke_setup_url: /kubernetes-engine/docs/tutorials/serve-gemma-tpu-jetstream

(That's my guess as to the value. Please correct if wrong!!)